reference/docs-conceptual/Crescendo/whats-new/whats-new-in-crescendo-11.md

```diff
@@ -162,25 +175,35 @@ Example: Calling a script based transformation.
 ]
 ```
 
-## Installing Crescendo
-
-Requirements:
-
-- **Microsoft.PowerShell.Crescendo** requires PowerShell 7.0 or higher
+## Creating cmdlet parameters that are not used by the native command
 
-To install **Microsoft.PowerShell.Crescendo** using the new **PowerShellGet** v2.x:
+Crescendo is designed to pass parameters defined in the configuration as arguments to the native
+application. There are times when you may want to use a parameter value in the output handler but
+not the native application. To enable this, a new boolean parameter property `ExcludeAsArgument` set
+to true prevents the argument from being sent to the native application. The default is `false`.
 
-```powershell
-Install-Module -Name Microsoft.PowerShell.Crescendo -AllowPreRelease
+```json
+"Parameters": [
+        {
+            "Name": "Filter",
+            "ParameterType": "string",
+            "ExcludeAsArgument": true,
+            "Mandatory": false,
+            "Description": "Variable not sent to native app"
+        }
+    ],
 ```
 
-To install **Microsoft.PowerShell.Crescendo** using the new [PowerShellGet v3][03]:
+For this example, the string argument passed to the **Filter** parameter is stored in the variable
+`$Filter`. This variable is available for use in your output handler. The `$Filter` variable could
+used by the output handler to filter the output of the native command, rather than returning all
+output.
```

Comment:
```suggestion
For this example, the string argument passed to the **Filter** parameter is stored in the variable
`$Filter`. This variable is available for use in your output handler. The output handler could use
the value of the `$Filter` variable to filter the output of the native command, rather than
returning all output.
```